src/shaders/terrain_raster.vertex.glsl

```diff
@@ -7,11 +7,15 @@ attribute vec2 a_texture_pos;
 varying vec2 v_pos0;
 
 const float skirtOffset = 24575.0;
+const float wireframeOffset = 0.00015;
 
 void main() {
     v_pos0 = a_texture_pos / 8192.0;
     float skirt = float(a_pos.x >= skirtOffset);
     float elevation = elevation(a_texture_pos) - skirt * u_skirt_height;
+#ifdef TERRAIN_WIREFRAME
+    elevation += u_skirt_height * u_skirt_height * wireframeOffset;
```

Comment:
This line is totally ad-hoc, really to bring in a necessary factor of the zoom (via skirt_height) so that it avoids z-fighting, but to avoid an extra uniform just for wireframe debugging.

As noted, polygon offset of the terrain would be a "more correct" solution, but I'm not sure that's worthwhile for a low consequence, probably rarely used debug feature.